CMakeLists.txt

```diff
@@ -47,8 +51,4 @@ foreach (GLSL ${GLSL_SOURCE_FILES})
     list(APPEND SPIRV_BINARY_FILES ${SPIRV})
 endforeach (GLSL)
 
-add_custom_target(
-    shaders
-    DEPENDS ${SPIRV_BINARY_FILES}
-)
```

Comment:
Why was this target removed? Now add_dependencies will throw errors because there is no shaders target.